### ISSUE TITLE
[Bug] Only 5 bits are displayed in tooltip for bitfields 

### ISSUE THREAD

**Issue by Nintynuts** [WerWolv/ImHex#1075]
### Operating System

Windows

### What's the issue you encountered?

When hovering over a byte with a bitfield pattern applied, only 5 bits are shown (and not just the set bits).

### How can the issue be reproduced?

Just hover over the hex for this example while the pattern is applied.

test pattern:
```
bitfield Test{
    a0:1;
    a1:1;
    a2:1;
    a3:1;
    a4:1;
    a5:1;
    a6:1;
    a7:1;
};

Test test[35] @ 0x0;
```

test hex:
```
01 02 04 08 10 20 40 80 03 05 09 11 21 41 81 06 0A 12 22 42 82 0C 14 24 84 18 28 48 88 30 50 90 60 A0 C0
```

### ImHex Version

1.28.0

### ImHex Build Type

- [X] Nightly or built from sources

### Installation type

MSI

**Comment by paxcut** [comment]
Please try the latest nightly build (link at the bottom of [https://imhex.werwolv.net/](url) ) because I think this was recently fixed. The bug existed in my previous nightly (5 days old) but my local up-to-date build and the latest msi installed nightly show all 8 bits on every byte when hovering it.

**Comment by Nintynuts** [comment]
Yep, they're all there in the latest nightly, thanks. It would be nice if the value data was aligned as a column, but it is usable.
